js/pages/cohort-definitions/components/reporting/cohort-reports/const.js

```diff
@@ -1,10 +1,13 @@
 define([], () => {
+    const COHORT_REPORT_COMPONENT_TYPE = 'atlas-cohort-report';
```

Comment:
related to the other comment about establishing constants: should we group all the plugin type constants together, such that anyone who wants to register their own component for the plugin, would do something like `registry.add(pluginConst.COHORT_REPORT_COMPONENT,{...pluginDef...})`.

My point is I don't think the constant for cohort report or profile widget belongs to the cohort-reports/profile-manager components...those should refer to the same constants that were used to register the plugins, which should belong to a pluginConstants module or something like that.